job-dsls/jobs/osbs/operators_osbs_seed_cloud_jobs.groovy

```diff
@@ -0,0 +1,207 @@
+                                  "CEKIT_CACHE_LOCAL": [CEKIT_CACHE_LOCAL, '-l'],
+                          ]
+
+                          def OPTIONAL_BUILD_SWITCHES = [
+                                  "VERBOSE": [VERBOSE, '-g'],
+                          ]
+
+       
+                          // The build script is in the image, but build.sh and build-overrides.sh which it calls will be downloaded
+                          def build_command = "build-osbs.sh"
+                         
+                          // Create the build command
+                          validateParameters(REQUIRED_BUILD_PARAMETERS, OPTIONAL_BUILD_PARAMETERS)
+
+                          for(param in REQUIRED_BUILD_PARAMETERS){
+                              def arg = param.value[0]
+                              def flag = param.value[1]
+                              build_command += " ${flag} ${arg}"
+                          }
+
+                          for(param in OPTIONAL_BUILD_PARAMETERS){
+                              def arg = param.value[0]
+                              def flag = param.value[1]
+                              if(arg) build_command+= " ${flag} ${arg}"
+                          }
+
+                          for(param in OPTIONAL_BUILD_SWITCHES){
+                              def arg = param.value[0]
+                              def flag = param.value[1]
+                              if(arg == "true") build_command+= " ${flag}"
+                          }
+
+                          build_command +=" -w ${WORKSPACE}"
+                          
+                          operator_repo_name = getOperatorRepoName(PROD_COMPONENT)
+                          
+                          operator_branch = getOperatorBranch(PROD_COMPONENT)
+                          
+                          // Checking out from git the Operator repo
+                          sh "git config --global user.email $GIT_EMAIL"
+                          sh "git config --global user.name $GIT_AUTHOR"
+                          githubscm.getRepositoryScm(operator_repo_name, GITHUB_ORG_UNIT, operator_branch)
+                          
+                          // Run the build script that should be into the operator hack folder
+                          dir("hack") {
+                            sh "source ~/virtenvs/cekit/bin/activate && $build_command"
+                          }
+                          // post processing
+                      }
+
+                      node("osbs-builder"){
+                          stage("building ${PROD_COMPONENT}") {
+                              ws {
+                                  // Add the working directory to the current path so any scripts dropped there are on the path
+                                  withEnv(["PATH+W=$WORKSPACE"]) {
+                                      sh 'rm -rf ${WORKSPACE}/{*,.*} || true'
```

Comment:
```suggestion
                                      sh "rm -rf ${WORKSPACE}/{*,.*} || true"
```
you replace from groovy to replace ` ${WORKSPACE}` right?

Reply:
yes